### PR TITLE
[techsupport] Added additional validation for test case "test_techsupport"

### PR DESCRIPTION
**Pull request by ppikh**
Signed-off-by: Petro Pikh <petrop@nvidia.com>

### Description of PR
Added additional validation for test case "test_techsupport"

Now test in additional doing check for content of techsupport dump file
- Check that we have "sai_sdk_dump" folder in tar.gz file and folder not empty
- Check that we have "dump" folder in tar.gz file and folder has not less than 50 files
- Check that we have "etc" folder in tar.gz file and folder has not less than 50 files
- Check that we have "log" fodler in tar.gz file and folder not empty

Summary: Added additional validation for test case "test_techsupport"
Fixes # (issue) fixed issue when techsupport has empty "dump" or "logs" or other folders but test passed

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [x] 202205

#### What is the motivation for this PR?
Added possibility to handle case when techsupport dump has empty "dump" or "log" or other folder. Previously test passed, not test will check that folders not empty.

#### How did you do it?
Added validation for folders which available in dump file

#### How did you verify/test it?
Executed all tests from file

### CODE REVIEW

**Comment by liat-grozovik** [comment]
@roy-sror could you please help to review?

**Review by roy-sror — DISMISSED** [review]

**Comment by ppikh** [comment]
@liat-grozovik fixed code style issues(pre-check)

File: tests/show_techsupport/test_techsupport.py [hunk 1]
```diff
@@ -317,14 +322,43 @@ def test_techsupport(request, config, duthosts, enum_rand_one_per_hwsku_frontend
...
+    sai_sdk_dump = duthost.command("ls {}/sai_sdk_dump/".format(dump_folder_path))["stdout_lines"]
+    dump = duthost.command("ls {}/dump/".format(dump_folder_path))["stdout_lines"]
+    etc = duthost.command("ls {}/etc/".format(dump_folder_path))["stdout_lines"]
+    log = duthost.command("ls {}/log/".format(dump_folder_path))["stdout_lines"]
+
+    assert len(sai_sdk_dump), "Folder 'sai_sdk_dump' in dump archive is empty. Expected not empty folder"
+    assert len(dump) > MIN_FILES_NUM, "Seems like not all expected files available in 'dump' folder in dump archive. " \
+                                      "Test expects not less than 50 files. Available files: {}".format(dump)
+    assert len(etc) > MIN_FILES_NUM, "Seems like not all expected files available in 'etc' folder in dump archive. " \
+                                     "Test expects not less than 50 files. Available files: {}".format(etc)
```
Comment (yxieca) [inline tests/show_techsupport/test_techsupport.py:355]: Why do we want to have this assertion?

Comment (yxieca) [reply tests/show_techsupport/test_techsupport.py:355]: Never mind. show tech was grabbing only a few files.

**Review by yxieca — APPROVED** [review]

**Pull request merged**